ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -173,12 +173,12 @@
 _PATH_FUS_110 = "fus_110.bin"
 _PATH_WS_BLE_HCI = "ws_ble_hci.bin"
 
-# This address is correct for versions up to v1.8 (assuming existing firmware deleted).
+# This address is correct for versions up to v1.12.1 (assuming existing firmware deleted).
 # Note any address from the end of the filesystem to the SFSA would be fine, but if
 # the FUS is fixed in the future to use the specified address then these are the "correct"
 # ones.
 _ADDR_FUS = 0x080EC000
-_ADDR_WS_BLE_HCI = 0x080DC000
+_ADDR_WS_BLE_HCI = 0x080E1000
```

Comment:
I think this line should remain unchanged.  Increasing the value may mean that it no longer works on early versions like v1.8.  The existing value should work on newer versions.

Reply:
This change was not included in the merged commit.